Python Program to Find LCM.py

```diff
@@ -1,20 +1,21 @@
 # Python Program to find the L.C.M. of two input number
 
+
 def compute_lcm(x, y):
+    # choose the greater number
+    if x > y:
```

Comment:
In this block I would ignore the `else` statement by initializing the variable with a default value first then just use an `if-statement` as follows:

```python
greater = y
if x > y:
    greater = x
```
its ideal to choose the default value which in most cases would be true like 

```python
n = input("Enter you name: ")
err = False
if not n:
    # if n is empty
    err = True
``` 

I assumed the input is going to be true by saying `err = False`
because in most cases the `n` is going to be entered by the user, there is no way the user can bypass to the next statement without entering the input unless its empty, therefore, `err=False`